### ISSUE TITLE
Object element is missing text

### ISSUE THREAD

**Issue by katmeikle** [wet-boew/GCWeb#1304]
Hi, we are in the process of installing the WET4.026 version and have encountered an accessibility error within the template. I know this issue was brought up twice before (https://github.com/wet-boew/wet-boew/issues/4699 & https://github.com/wet-boew/GCWeb/issues/816) without being properly addressed. 

The issue is the object element is missing text. I’ve been told that because the object element sits inside the <a href> then the text associated to the link satisfies the object. However, we’ve marked up some scenarios and while this implementation will satisfy a non-sighted user it will not work for a sighted user if something happens to the image in the object element, such as a loading issue or missing image. Please see the example mock-ups on the following links:

http://www.tpsgc-pwgsc.gc.ca/123test/object_test/broken_object_with_text.html 
http://www.tpsgc-pwgsc.gc.ca/123test/object_test/ok_object_with_text.html 
http://www.tpsgc-pwgsc.gc.ca/123test/object_test/broken_object.html 

We’ve examined the code and making the fix is as simple as moving the span tag inside the object. 
Curent code: 
<a href="https://www.canada.ca/en.html">
<object type="image/svg+xml" tabindex="-1" data="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_26/assets/sig-blk-en.svg"></object>
<span class="wb-inv">Government of Canada / 
<span lang="fr">Gouvernement du Canada</span>
</span>
</a>

Fixed code: 
<a href="https://www.canada.ca/en.html">
<object type="image/svg+xml" tabindex="-1" data="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_26/assets/sig-blk-en.svg"><span>
Government of Canada / <span lang="fr">Gouvernement du Canada</span>
</span>
</object>
<span class="wb-inv">Government of Canada / 
<span lang="fr">Gouvernement du Canada</span>
</span>
</a>

We know that making this simple change could potentially cause a stutter for some screen reader users however this implementation will better satisfy the WCAG requirement and ensure a more consistent user experience for all.

**Comment by katmeikle** [comment]
Hi just wondering if anyone has any feedback about this issue and fix?

Thanks

**Comment by nschonni** [comment]
Can you explain what you think the accessibility issue is? There was discussion and resolutions in the issues you linked to.
Also, if the browser doesn't support SVG, the tag gets replaced with an image with and empty alt tag (as it's presentational) https://github.com/wet-boew/wet-boew/blob/f5f33e0240649fe22fd12f28978113841762bffa/src/polyfills/svg/svg.js
Moving the `span` inside appears to go agains the spec https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object, but adding the `role=img` might be appropriate now

**Comment by katmeikle** [comment]
The accessibility issue is if the image if missing or can't be accessed the screen reader will read it but a sighted user will not see the image.

**Comment by jkshapiro** [comment]
Also please see https://dequeuniversity.com/rules/axe/2.6/object-alt

Even if there is no real issue here, as suggested at #816, this omission causes errors to be reported by automated testing tools. This adds noise, making a11y testing more difficult and increasing the chance that testers will miss real issues.

**Comment by jkshapiro** [comment]
Another solution would be to add an ID to the span and use aria-labelledby to refer to it.

**Comment by nschonni** [comment]
The object wrapping is being removed from this theme with #1329

**Comment by zzucco** [comment]
With the above PR, is this still an issue?
